README.md

```diff
@@ -164,6 +164,7 @@ It's good for development, but this build can be in 100+ times slower for some m
 If you would like to analyse CPU performance of libindy for your use case, you have to use release artifacts (`cargo build --release`). 
 
 ## How to start local nodes pool with docker
+THESE INSTRUCTIONS ARE NO LONGER VALID
```

Comment:
Right now it's main and recommended way to obtain local development pool. Why do you like to mark it as invalid?

Reply:
It's better to put the link to this section ( `
https://github.com/hyperledger/indy-sdk/blob/master/README.md#how-to-start-local-nodes-pool-with-docker ` ) to other places (e.g. RHEL readme https://github.com/hyperledger/indy-sdk/blob/master/doc/rhel-build.md#starting-up )